### ISSUE TITLE
Program O 2.4.8 is now available. Click here to download it.

### ISSUE THREAD

**Issue by BeZazz** [Program-O/Program-O#111]
I just did  git pull and now in the admin section at the top left I am seeing

> Program O 2.4.8 is now available. Click here to download it.

_(I would swear it was not there before)_

**Comment by Program-O** [comment]
thats strange.
that message should appear when your version is out of date
and it should refer to the most recent version
which should be ... 2.5.1
can you look in the contents of version.txt and let me know what it says

**Comment by Program-O** [comment]
it seems that is the version on the dev branch
have you pulled from master or dev?

**Comment by BeZazz** [comment]
Thought I pulled master will check.
I know in the version file it has
2.5.1

**Comment by BeZazz** [comment]
/usr/local/cpanel/3rdparty/bin/git branch
* master

**Comment by Program-O** [comment]
what is the output of 
git remote show origin

**Comment by BeZazz** [comment]
sbotbezazz@jupiter [~/public_html]# /usr/local/cpanel/3rdparty/bin/git remote show origin
* remote origin
  Fetch URL: https://github.com/Program-O/Program-O.git
  Push  URL: https://github.com/Program-O/Program-O.git
  HEAD branch: master
  Remote branches:
    dev    tracked
    master tracked
  Local branch configured for 'git pull':
    master merges with remote master
  Local ref configured for 'git push':
    master pushes to master (up to date)

**Comment by Program-O** [comment]
everything looks ok
i am stumped and will have to think about it some more
i will update the dev repo and see if that does anything

**Comment by Program-O** [comment]
can you pull again?
i think i know what caused it

**Comment by BeZazz** [comment]
I just did a git pull and tried a few hard refreshes but it is still there.

**Comment by Program-O** [comment]
and still no response from the bot?
i thought that by moving my config around it might have rewritten the config file require paths
but i was wrong
are your bot debug files revealing anything?

**Comment by BeZazz** [comment]
In the admin section it is stuck in
"learning new answers" mode

Where a "client" would access the bot the only one to choose is "Unknown"

**Comment by Program-O** [comment]
ok i think if we tackle the weird as there is definitely something weird going on.
would you be willing to pull down a completely new copy to a new location to see if that gets rid of error message?

if you are cool the that then...
first try with a new database and see if that solves the problem with the stray version number
if that fixes it drop your old global_config.php over the freshly installed one and see if everything is happy at that point

**Comment by BeZazz** [comment]
Just before I do that I noticed your bot at 
http://www.program-o.com/

Is also doing the 
Yes or No?
thing.

**Comment by Program-O** [comment]
:( you are right i will investigate

**Comment by Program-O** [comment]
i have isolate the issue 
hopefully it wont take long to fix

**Comment by Dave-Morton** [comment]
Have you tried logging out of the admin page, then logging back in?

**Comment by BeZazz** [comment]
Just tried that, now I'm seeing
_Program O 2.5.2 is now available. Click here to download it._
version.txt contains 2.5.2

**Comment by Dave-Morton** [comment]
Interesting... I assume you've cleared your browser cache and such. I'm simply not seeing the issue here on my Windows 10/Firefox setup.

**Comment by Program-O** [comment]
i have fixed the issue with they YES/NO please pull and confirm it fixes the issue your side

**Comment by Program-O** [comment]
the version issue i am guessing is some weird caching thing.
when if you refresh before you pull you should see Program O 2.5.3 is now available. Click here to download it.
after you pull that should go
if not and you have all the correct version.txt and everything else works correctly it must be a caching thing

**Comment by BeZazz** [comment]
I had to logout then back in to get 
Program O 2.5.2 is now available. Click here to download it.
to change to
Program O 2.5.3 is now available. Click here to download it.

I did a it pull then logged out and back in.
I now see
Program O version 2.5.3

Bot now answers correctly again in admin but still only have the option for Unknown on the main page.
Which I will look into on my side.


Thanks for all the help :)
